OrbitClientGgp/ClientGgp.cpp

```diff
@@ -147,35 +137,28 @@ bool ClientGgp::LoadModuleAndSymbols() {
   // Load symbols for the module
   const std::string& module_path = module->m_FullName;
   LOG("Looking for debug info file for %s", module_path);
-  ErrorMessageOr<std::string> result_main_executable_debug_file =
-      process_client_->FindDebugInfoFile(module_path);
-  if (result_main_executable_debug_file.has_error()) {
-    ERROR("Error loading symbols: %s", result_main_executable_debug_file.error().message());
-    return false;
-  }
-  const std::string& main_executable_debug_file = result_main_executable_debug_file.value();
+  OUTCOME_TRY(main_executable_debug_file, process_client_->FindDebugInfoFile(module_path));
   LOG("Found file: %s", main_executable_debug_file);
   LOG("Loading symbols");
-  ErrorMessageOr<orbit_grpc_protos::ModuleSymbols> result_symbols =
-      symbol_helper_.LoadSymbolsFromFile(main_executable_debug_file, module->m_DebugSignature);
-  if (result_symbols.has_error()) {
-    ERROR("Error loading symbols: %s", result_symbols.error().message());
-    return false;
-  }
-  module->LoadSymbols(result_symbols.value());
+  OUTCOME_TRY(symbols, symbol_helper_.LoadSymbolsFromFile(main_executable_debug_file,
+                                                          module->m_DebugSignature));
+  module->LoadSymbols(symbols);
   target_process_->AddFunctions(module->m_Pdb->GetFunctions());
-  return true;
+  return outcome::success();
 }
 
 bool ClientGgp::InitCapture() {
-  target_process_ = GetOrbitProcessByPid(options_.capture_pid);
-  if (target_process_ == nullptr) {
-    ERROR("Error: not able to set target process");
+  ErrorMessageOr<std::shared_ptr<Process>> target_process_result =
+      GetOrbitProcessByPid(options_.capture_pid);
+  if (target_process_result.has_error()) {
```

Comment:
Nit: just so you know, the check `if(!target_process_result)` would also work. Probably not worth the change, but just in case you did not know.

Reply:
Ah, I didn't know, thanks!